editor.planx.uk/src/pages/FlowEditor/components/FlowEditor/ToggleHelpTextButton.tsx

```diff
@@ -0,0 +1,35 @@
+import HelpTextOffIcon from "@mui/icons-material/DoNotDisturbOff";
+import HelpTextIcon from "@mui/icons-material/Help";
+import IconButton from "@mui/material/IconButton";
+import Tooltip from "@mui/material/Tooltip";
+import { useStore } from "pages/FlowEditor/lib/store";
+import React from "react";
+
+export const ToggleHelpTextButton: React.FC = () => {
+  const [showHelpText, toggleShowHelpText] = useStore((state) => [
+    state.showHelpText,
+    state.toggleShowHelpText,
+  ]);
+
+  return (
+    <Tooltip title="Toggle help text" placement="right">
+      <IconButton
+        aria-label="Toggle help text"
+        onClick={toggleShowHelpText}
+        size="large"
+        sx={(theme) => ({
+          background: theme.palette.background.paper,
+          padding: theme.spacing(1),
+          color: showHelpText
+            ? theme.palette.text.primary
+            : theme.palette.text.disabled,
+          "&:hover": {
+            background: theme.palette.common.white,
+          },
+        })}
+      >
+        {showHelpText ? <HelpTextIcon /> : <HelpTextOffIcon />}
```

Comment:
We're running out of options for pure "on"/"off" pairs - but I think these two work alright together?

![Screenshot from 2024-12-01 20-48-54](https://github.com/user-attachments/assets/b31ee650-e217-4ba7-804d-bf7524357308) ![Screenshot from 2024-12-01 20-49-00](https://github.com/user-attachments/assets/1260ffcd-e106-4679-bb33-0aa544da2738)

Other suggestions welcome if not obvious enough: https://mui.com/material-ui/material-icons/ ("speaker notes" was my second vote but quite disconnected from "help" text specifically)